src/backend/main.ts

```diff
@@ -973,6 +978,24 @@ ipcMain.handle(
 
     logInfo(`Launching ${title} (${game.app_name})`, LogPrefix.Backend)
 
+    if (autoSyncSaves && isOnline()) {
```

Comment:
Shoudl this not be disabled for sideload games?

Reply:
But there is any way to activate this setting on sideloaded games? I don't think so.